media/css/mozorg/contribute/contribute-2015.less

```diff
@@ -956,6 +981,77 @@ textarea:focus {
         margin: 0;
         padding: 0;
     }
+
+    .signup-option {
+        font-size: 1.5em;
+        text-align: center;
+        padding-bottom: 20px;
+    }
+}
+
+.inquiry-choices {
+    ul {
+        list-style: none;
+    }
+
+    a {
+        position: relative;
+        display: block;
+
+        &:before {
```

Comment:
Instead of duplicating all this CSS it'll be simpler to hang these icons from a class we can apply equally to links or labels, thought that will mean also adding the class to the current form as well.